src/explore-education-statistics-admin/src/prototypes/components/PrototypePreReleaseCreate.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react';
+import FormEditor from '@admin/components/form/FormEditor';
+import SanitizeHtml from '@common/components/SanitizeHtml';
+
+const CreatePreRelease = () => {
+  // const query = new URLSearchParams(window.location.search);
+  // const dialog = query.has('showDialog');
+
+  const [createPreRelease, setCreatePreRelease] = useState(true);
+  const [addNewPreRelease, setAddNewPreRelease] = useState(false);
+  const [previewPreRelease, setPreviewPreRelease] = useState(false);
+  const [editPreRelease, setEditPreRelease] = useState(false);
+
+  const formIntro = `
+    <p>Beside Department for Education (DfE) professional and production staff the 
+    following post holders are given pre-release access up to 24 hours before release.</p>
+
+    <ul>
+        <li>ADD ROLES HERE</li>
+    </ul>
+`;
+
+  const formText = `
```

Comment:
What's the motivation for this being free text rather than an actual list (similar to user invites)